code401challenges/src/test/java/code401challenges/utilities/MultiBracketValidationTest.java

```diff
@@ -1,4 +1,29 @@
 package code401challenges.utilities;
 
+import org.junit.Test;
+
+import static code401challenges.utilities.MultiBracketValidation.multiBracketValidation;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
 public class MultiBracketValidationTest {
+// HAPPY TEST
+    @Test public void allCorrectOrderTest(){
+        String input = "{}{Code}[Fellows](())";
+        boolean actual = multiBracketValidation(input);
+        assertTrue(actual);
+    }
+// FAIL CASE
+    @Test public void failCaseTest(){
+        String input = "(";
+        boolean actual = multiBracketValidation(input);
+        System.out.println(actual);
+        assertFalse(actual);
+    }
```

Comment:
This whole test could really just be a one liner: `assertFalse(multiBracketValidation("("));`